cmd/security-agent/app/app.go

```diff
@@ -89,7 +91,9 @@ Datadog Security Agent takes care of running compliance and security checks.`,
 )
 
 func init() {
-	SecurityAgentCmd.PersistentFlags().StringArrayVarP(&confPathArray, "cfgpath", "c", nil, "path to a yaml configuration file")
+	var defaultConfPathArray = []string{path.Join(commonagent.DefaultConfPath, "datadog.yaml"),
```

Comment:
just to confirm? what should be the order for the configuration file?
If `datadog.yaml` is present, the security-agent is not looking for `security-agent.yaml`? is is the opposite?

Reply:
The security-agent will open all configuration files specified with `-c`, and will merge their configuration parameters.

By default, when no `-c` are specified, both file will be opened, one after another.